### PR TITLE
Use camelCase cli arguments internally

### PR DESCRIPTION
**Pull request by fisker**
## Description

I think it's easier to use.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/cli/options/parse-cli-arguments.js [hunk 1]
```diff
@@ -18,7 +19,14 @@ function parseArgv(rawArguments, detailedOptions, logger, keys) {
...
 
-  return normalizeCliOptions(argv, detailedOptions, { logger });
+  const normalized = normalizeCliOptions(argv, detailedOptions, { logger });
+
+  return Object.fromEntries(
+    Object.entries(normalized).map(([key, value]) => {
+      const option = detailedOptions.find(({ name }) => name === key) || {};
+      return [option.forwardToApi || camelCase(key), value];
+    })
+  );
```
Comment (sosukesuzuki) [inline src/cli/options/parse-cli-arguments.js:29]: It looks hard to understand the purpose of the code. Can you add a comment? For example, `Make cli options camelCase for readability`

**Review by sosukesuzuki — COMMENTED** [review]
Looks good, please check one comment

File: src/cli/options/parse-cli-arguments.js (hunk 1 above)
Comment (fisker) [reply src/cli/options/parse-cli-arguments.js]: https://github.com/prettier/prettier/pull/11976/commits/45601cff4020fb1adcabf70ce6d26ef95ea79186

**Review by sosukesuzuki — APPROVED** [review]
Thanks

**Pull request merged**